classes/Cart.php

```diff
@@ -766,30 +766,30 @@ public function getProducts($refresh = false, $id_product = false, $id_country =
         $sql->leftJoin('image_lang', 'il', 'il.`id_image` = image_shop.`id_image` AND il.`id_lang` = ' . (int) $this->getAssociatedLanguage()->getId());
 
         /** @var array<string, mixed> $result */
```

Comment:
```suggestion
        /** @var array<string, mixed> $products */
```
As mentioned by phpstan

Reply:
Fixed!